packages/playground/src/components/branches-list.tsx

```diff
@@ -20,24 +35,50 @@ interface BranchProps {
   }
   pullRequest: {
     sha: string
-    status: string
+    // status: string
   }
 }
 
 interface PageProps {
   branches: BranchProps[]
 }
 
-export default function BranchesList({ branches }: PageProps) {
+const CopyButton = () => {
+  const [copied, setCopied] = useState(false)
+
+  useEffect(() => {
+    let timeoutId: number
+    if (copied) {
+      //add copy function here if we need in the future
+      timeoutId = window.setTimeout(() => setCopied(false), 2500)
+    }
+    return () => {
+      clearTimeout(timeoutId)
+    }
+  }, [copied])
+
+  return (
+    <Button variant="ghost" size="xs" onClick={() => setCopied(true)}>
```

Comment:
What does it copy right now? It should copy Branch name to clipboard on copy.

Reply:
copy the branch name currently